.github/workflows/binary-applications.yml

```diff
@@ -140,7 +160,9 @@ jobs:
             PYTHON_ARCH: x86
         steps:
             - uses: actions/checkout@v2
-            - run: git fetch --prune --unshallow
+              with:
+                  fetch-depth: 0  # fetch complete history
+            - run: git fetch origin +refs/tags/*:refs/tags/*
 
             - name: Fetch git tags
               run: git fetch origin +refs/tags/*:refs/tags/*

```

Comment:
Same as previous job, an extra git fetch?

Reply:
I see this pattern repeated elsewhere too, so probably there is a reason? I'll stop commenting on every occurrence.

Reply:
Hmm this duplication is either sloppy copy-pasting on my part or a merge is putting that line in there.  Hopefully it's the former.  In any case, patching this in 61e7b794.